### ISSUE TITLE
Minting 3.0 requirements definition

### ISSUE THREAD

**Issue by weynandkuijpers** [threefoldtech/tfchain_pallets#71]
# Minting V3.0 working and requirements.

- [ ] [Farming rewards 3.0 definition](https://library.threefold.me/info/threefold#/tfgrid/farming/threefold__farming_reward).
```
TFT earned per month per node = 
   ( + CU farmed * CU farming reward 
    + SU farmed * SU farming reward 
    + NU used * NU farming reward
    + IPAddr used * IPAddr farming reward ) * multiplier
```
- [ ] Formulas to be used are below.  They come  form the calculator and all other publications need to reflect these fomulas.
```
# CU=min((MRU-1)/4, CRU*2)
# SU=HRU/1200 + (SRU/300)(0.8
# NU=max(measured GB's in, measured GB's out)
# IPAddr = measured amount of hours of IPv4 used 
```
- [ ] Rewards at time of writing:

Unit | description | rewards
-- | -- | -- 
Compute Unit (CU) | typically 2 vcpu, 4 GB mem, 50 GB storage | 51.81 TFT/month 
Storage Unit (SU) | typically 1 TB of netto usable storage | 32.38 TFT/month
Network Unit (NU) | 1 GB of data transferred as used by TFGrid user for Public IP Addr | 0.65 TFT/GB
Public IPv4 Address | Public IP Address as used by a TFGrid user | 0.11 TFT/hour

- [ ]  Certified farming (nodes) get 25% more tokens as a reward

|  | DIY | Certified |
| -- | -- | -- |
| multiplier | 1 | 1.25 |

- [ ] TFT value pegged to USD (pricing changes in line with TFT/USD rate).  This is the value number used to calculate how much TFT's a CU/SU/NU/IPv4 gets as a proof of capacity reward.
     - [ ] current TFT to USD value is 0.05 , calculated on 2021-10-08 05:46:11
     - [ ] TFT value will be fixed for 1 month
     - [ ] TFT value will be calculated as an average over the last month.
          - [ ] Investigate and agree whether a longer period is required to avoid / counter manipulation by token holders.
     - [ ] Token value information comes from Liquid and coinmarket cap
     - [ ]  The reward TFT's for CU/SU/NU/IPv4 are locked for 60 months once farming starts for a 3node (first boot & registration to blockchain) or at date of purchase when certified 3node.
     - [ ] If a farmer decides to keep 3nodes online longer that the 60 month period a new TFT price for CU/SU/NU/IPv4 will be set for new period of 60 months.
          - [ ] This needs to be done automatically, without (application) service interruption
          - [ ] Minimal SLA’s need to be achieved before the farming reward can be earned (uptime, bandwidth, latency, …)
             - [ ] Minimal SLA for Europe is: 99%
             - [ ] Minimal SLA for North America is: 99%
             - [ ] Minimal SLA for South America is: ?
             - [ ] Minimal SLA for Africa is: ?
             - [ ] Minimal SLA for Australia is: 99% 
             - [ ] Minimal SLA for Asia is: 99%
             - [ ] Minimal SLA for South East Asia is: ?
- [ ] All farming reward tokens are locked.  Tokens are released when 30% of the 3node capacity is being utilized.  This 30% check is done once, once the farmers 3nodes has reached 30% utilization all historic tokens are released and all future farming reward tokens are no longer locked up.
- [ ] On TF Grid3.0 all farmers have registered there nodes on the TF Chain (polkadot/substrate).  All farming rewards should be paid to substrate accounts. However there are two options to mint:
     - [ ] Minting 3.0 remains operational on the Stellar network.  
     - [ ] Minting 3.0 starts on the TF Chain.
     - [ ] Minting 2.0 stays on the Stellar network and minting 3.0 commences on the TF Chain.

**Comment by andhartl** [comment]
that's not true: The reward TFT's for CU/SU/NU/IPv4 are locked for 60 months once farming starts for a 3node (first boot & registration to blockchain) or at date of purchase when certified 3node.

TFT minted each month are locked until 30% of the node's capacity are used or after 24 month.

**Comment by LeeSmet** [comment]
Duplicate of #48?

**Comment by weynandkuijpers** [comment]
#48 is not a spec sheet - that a discussion. I meant this to be a spec sheet and spec sheet only.

**Comment by weynandkuijpers** [comment]
> certified: SLA is 99.5% for datacenter farmers and 97% for home and office farmers diy: suggest 90% details: https://library.threefold.me/info/threefold#/tfgrid/farming/threefold__farming_certified_requirements

Implemented in the top spec list.

**Comment by weynandkuijpers** [comment]
> that's not true: The reward TFT's for CU/SU/NU/IPv4 are locked for 60 months once farming starts for a 3node (first boot & registration to blockchain) or at date of purchase when certified 3node.
> 
> TFT minted each month are locked until 30% of the node's capacity are used or after 24 month.

Implemented comment in top requirements list.
